x-pack/plugins/cloud_security_posture/server/saved_objects/cis_1_4_1/csp_rule_template.ts

```diff
@@ -0,0 +1,29 @@
+/*
```

Comment:
This file should probably not be in the `cis_1_4_1` directory, but direclty under `saved_objects`